tests/tests/conus13km_control

```diff
@@ -0,0 +1,29 @@
+###############################################################################
+#
+#  HRRR physics on 13km domain, control
```

Comment:
Just want to confirm, these conus 13km HRR tests are actually runs on global domain, not on conus domain, correct?

Reply:
The tests with "conus13km" in their name run on a 13km resolution CONUS domain. Not global. That is why they have "conus" and "13km" in their name.

Reply:
@junwang-noaa @BrianCurtis-NOAA is the conversation good to be resolved?

Reply:
Yes